packages/app/package.json

```diff
@@ -70,6 +70,7 @@
     "@roadiehq/backstage-plugin-github-insights": "^2.0.0",
     "@roadiehq/backstage-plugin-github-pull-requests": "^2.0.0",
     "@roadiehq/backstage-plugin-travis-ci": "^2.0.0",
+    "@internal/plugin-catalog-customized": "1.2.1-next.1",
```

Comment:
Just for completeness, let's try removing the existing catalog plugin import as well